FlorenceBML/BMLacq38.xml

```diff
@@ -34,6 +35,376 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                    <p>Holes in <locus target="#79 #139"/>.</p>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf" quantity="262">4+145</measure>
+                                    <measure unit="leaf" type="blank">1</measure><locus target="#117v 145v"/>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>140</height>
+                                        <width>100</width>
+                                    </dimensions>
+                                </extent>
+                                <foliation>Folios numbered in pencil on the upper left, and then again, after partial
+                                    cancellation of previous numbering, on the bottom right, by recent European hands. 
+                                    Three unnumbered paper protection leaves and one unnumbered parchment protection leaf before the textblock.
+                                    as well as three unnumbered paper protection leaves at the end.</foliation> 
+                                <collation>
+                                    <list>
+                                        <item xml:id="q1" n="1">
+                                            <dim unit="leaf">4</dim>
+                                            <locus from="i" to="iii"/>
+                                            s.l. 3, no stub
+                                        </item>
+                                        <item xml:id="q2" n="2"> 
+                                            <dim unit="leaf">9</dim>
+                                            <locus from="iv" to="8"/>
+                                            s.l. 1, stub before 1 and after 9
+                                        </item>
+                                        <item xml:id="q3" n="3">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="9" to="16"/>
+                                        </item>
+                                        <item xml:id="q4" n="4">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="17" to="24"/>
+                                        </item>
+                                        <item xml:id="q5" n="5">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="25" to="32"/>
+                                        </item>
+                                        <item xml:id="q6" n="6">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="33" to="40"/>
+                                            s.l. 3 and 6, stub after 2 and 5
```

Comment:
does this "summarised" system of noting stubs (not individually) work with the app? If so, it could be added to the guidelines.

Reply:
I did it not for purpose! Is it better to encode in this way:   s.l. 3, stub after 5; s.l. 6, stub after 5

Reply:
That is the only way that I'm aware of (actually, "s.l." is not even needed, "3, stub after 5 6, stub after 5" is enough for the system)